### DISCUSSION TITLE
Numpy Array not Writable error

### DISCUSSION THREAD

**Discussion by nthevan** [tbepler/topaz#157] [General]
Hello, 

I've sucessfully installed Topaz in a conda environment. The conda environment uses python 3.10 to allow for the correct pytorch installation (I need CUDA11.6 compatibility). Unfortunately, I think this version of python doesn't have the correct numpy for Topaz. I'm trying to run Topaz from the wrapper in Relion4.0 and I'm stuck with this error: 

UserWarning: The given NumPy array is not writable, and PyTorch does not support non-writable tensors. This means writing to this tensor will result in undefined behavior. You may want to copy the array to protect its data or make it writable before converting it to a tensor. This type of warning will be suppressed for the rest of this program. (Triggered internally at  /opt/conda/conda-bld/pytorch_1659484803030/work/torch/csrc/utils/tensor_numpy.cpp:172.

Does anyone know how to fix for this? I'm new to both Linux and Relion.

**Comment by tbepler** [comment]
This warning shouldn't cause any problems, because we don't write these
arrays internally. It is annoying, though, so we'll look at suppressing it
in future releases.

If this is causing other errors or you're seeing unexpected problems with
the output, then let us know.
